src/langcheck/metrics/zh/reference_free_text_quality.py

```diff
@@ -90,9 +90,17 @@ def sentiment(
     global _sentiment_model_path
 
     _sentiment_pipeline = pipeline(
-        'sentiment-analysis', model=_sentiment_model_path
-    )  # type: ignore[reportGeneralTypeIssues]  # NOQA: E501
+        'sentiment-analysis',
+        model=_sentiment_model_path)  # type: ignore[reportGeneralTypeIssues]
     # {0:"Negative", 1:'Positive'}
+    from langcheck.metrics import _model_manager
+    tokenizer, model = _model_manager.fetch_model(lanaguage='zh',
+                                                  metric='sentiment')
+    _sentiment_pipeline = pipeline(
+        'sentiment-analysis',
+        model=model,  # type: ignore[reportGeneralTypeIssues]
+        tokenizer=tokenizer  # type: ignore[reportGeneralTypeIssues]
+    )
```

Comment:
I don't think this is working? If I understand correctly, we should be using the `load_pipeline_for_text_classification` function so that `fetch_model` will directly return a `pipeline` object

Reply:
> I don't think this is working? If I understand correctly, we should be using the `load_pipeline_for_text_classification` function so that `fetch_model` will directly return a `pipeline` object

I think using pipeline(model= model, tokenizer=tokenizer), would return the same result with the traditional way that call tokenizer() fisrt and then model(), becuase pipeline help us do this inside its class by preprosee(),_forward() and postprocess(). so the code here would not generate different result. May be we should require all people use the same inference way first.
![image](https://github.com/citadel-ai/langcheck/assets/59645193/5cf7b2cf-a314-4ac4-86d1-ffa327d3605e)

2. That's the point I wonder how to handle it. 

First, I check other language module, and pipeline is used in zh only. So, for current situation, a specific load_pipeline() seems redudant. I modify the code here in zh to make a load_auto_model() fit both pipeline inference way and traditional inference way, make loader become more simple.

------ Other findings when check your comment ------------
if our final goal is letting user modify model sets, unique postprocessing in text-classification seems impossible to be set in configuration. Because the number of class is different across models. Although, most of models may do binary classification task, but like toxicity model in zh, it has 5 class, I use a [1- sum(logits[1:5])] to transform logits to score, and in ja toxicity, use [ logits[0] + 0.5 * logits[1]] postprocessing to handle a 3 class text-classification. 
```
model_manager
  - __init__(): call from_file inside 
  - register(language, metric, model, tokenizer, reivsion, *args, **kwargs)
  - get_model(lang, metric, using_pipeline: bool):
  - from_file(file_name): use register(...) & validate(...) inside
  - validate(): check model availablity 
  - list_models(...)
  - save(): output conifg
```

Reply:
Thanks for the detailed comment! Those are some good points / questions, and definitely things we need to think about eventually. But for now, I think we can focus on just making `fetch_model` work for our existing models.

Basically I was running into some issues before, but a couple of them are fixed now after your commit (https://github.com/citadel-ai/langcheck/pull/73/commits/4433da1afc441ec6d9df8c254223358ec2620deb). I'm still running into a couple other issues with the `factual_consistency` model (I'll add a comment there directly)